config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "process.env.DB_USER",
+    "password": "process.env.DB_PASSWORD",
```

Comment:
The value for `password` is set as a string instead of referencing the environment variable. Consider using `process.env.DB_PASSWORD` directly without quotes to ensure the environment variable is used.